specification/gp-connect-user-permissions.yaml

```diff
@@ -502,7 +494,7 @@ components:
         message:
           type: string
           description: A message pertaining to the request and its status.
-          example: Request requires review by HCP.
+          example: Request requires review by a healthcare worker.
```

Comment:
this message example has a full stop and messages below (eg. lines 745 and 799) don't, not sure if this matters

Reply:
I didn't update it initially as I was going to keep the consistency between the example and the messages but I think worth updating as you noticed it too 😄